app/javascript/lib/validation/validate-file-content.js

```diff
@@ -177,9 +177,89 @@ function validateEqualCount(headers, annotTypes) {
   return issues
 }
 
+/**
+ * Verify "GENE" is present as the first column in the first row for an Expression Matrix file
+ * Todo: Accept files that are R-formatted as well via this ticket: SCP-
+ */
+function validateGeneInHeader(headers) {
+  const issues = []
+  if (headers[0][0].toUpperCase() !== 'GENE') {
+    const msg = 'Dense matrices require the first value of the file to be "GENE". ' +
+     `However, the first value for this file currently is "${headers[0][0]}".`
+    issues.push(['error', 'format:cap:missing-gene-column', msg])
+  }
+
+  return issues
+}
+
+
+/**
+ *
+ * Validate all values are numbers outside first column cell name
+ */
+function validateValuesAreNumeric(parsedLine, isLastLine, lineNum, dataObj) {
+  const issues = []
+
+  // skip first column
+  const lineWithoutFirstColumn = parsedLine.slice(1)
```

Comment:
Per rationale in my other comment beginning "Similar to".
```suggestion
function validateValuesAreNumeric(line, isLastLine, lineNum, dataObj) {
  const issues = []

  // skip first column
  const lineWithoutFirstColumn = line.slice(1)
```